.rubocop.yml

```diff
@@ -19,7 +19,7 @@ AllCops:
     - db/schema.rb
```

Comment:
Later versions of rubocop let you specify a cache folder. So we update rubocop as well to take advantage. It makes linting nice and fast if the rubocop yaml files haven't been touched.